index.html

```diff
@@ -1169,19 +1169,66 @@ <h2>Relative DID URLs</h2>
 
   </section>
 
-  <section class="informative">
+  <section>
     <h1>Data Model</h1>
-
+    <p>
+This specification defines an abstract data model for <a>DID documents</a> so it
+is independent of any specific representation. In this section we provide a
+high-level description of the data model, a set of requirements for
+representations, and a set of requirements for extensibility.
+    </p>
     <section>
       <h2>Definition</h2>
+      <p>
+A <a>DID document</a> consists of a collection of name-value pairs which will be
+referred to as properties in this document. The definitions of each of these
+properties are specified in section <a href="#core-properties"></a>. Specific
+representations are defined in section <a href="#core-representations"></a>.
+      </p>
     </section>
-
     <section>
-      <h2>Extensibility</h2>
+      <h2>Representations</h2>
+      <p>
+Following are the requirements for conformant representations.
+      </p>
+      <ol>
+        <li>
+A representation MUST define an unambiguous encoding and decoding of all
+properties defined in this specification such that the value of any property
+may be losslessly converted into another representation.
+        </li>
+        <li>
+The representation MUST have an unambiguous MIME type.
+        </li>
+        <li>
+The representation MUST define fragment processing rules for its MIME type that
+are conformant with this specification.
+        </li>
+      </ol>
     </section>
```

Comment:
```suggestion
        <p>
The core representations are specified in section <a href="#core-representations"></a>.
        </p>
    </section>
```

(I know it says this in the prior section too, but reading the Representations subsection alone, a mention of Core Representations is conspicuously absent.)